.pre-commit-config.yaml

```diff
@@ -25,6 +27,15 @@ repos:
         language: fail
         files: 'changelog/'
         exclude: '/(\d+\.[a-z]+(\.\d+)?\.rst|_template.rst.jinja|README.rst)$'
+      - id: spdx-licensing
+        name: SPDX License Identifiers
+        description: Check for the SPDX-License-Identifier in each file.
+        language: pygrep
+        entry: 'SPDX-License-Identifier: '
+        args: [--negate]
+        types: [text]
+        exclude_types: [json, pofile]
+        exclude: 'changelog/|py.typed|disnake/bin/COPYING|.github/PULL_REQUEST_TEMPLATE.md|LICENSE|MANIFEST.in'
```

Comment:
```suggestion
        exclude: 'changelog/|py.typed|disnake/bin/COPYING|.github/PULL_REQUEST_TEMPLATE.md|LICENSE|MANIFEST.in|requirements/|requirements.txt'
```

Reply:
why are those being excluded

Reply:
I don't see a point in adding license identifiers to requirement files, we don't need to overdo it. The `LICENSE` file always applies, regardless of the identifier in every file.

Reply:
I guess? Kinda goes against the purposes of SPDX but okay
https://github.com/DisnakeDev/disnake/pull/756/commits/8faa69d1336e1132ea383250b10b3be9e6c82f02